blogg/posts/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
```

Comment:
I don't think we'll need any models.  Can we remove this file?  Will Django complain?

Reply:
Can leave as discussed.  Otherwise, maybe we could use flask instead of Django for the test?